source/beamline/BioXAS/BioXASSideBeamline.h

```diff
@@ -35,6 +35,8 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 #include "beamline/CLS/CLSBasicCompositeScalerChannelDetector.h"
 #include "beamline/CLS/CLSMAXvMotor.h"
 #include "beamline/CLS/CLSJJSlit.h"
+#include "beamline/CLS/CLSJJSlitGapControl.h"
+#include "beamline/CLS/CLSJJSlitCenterControl.h"
```

Comment:
Hmmm both the Gap and Center controls actually inherit from a general JJSlitsBladesControl class--should that one be renamed?